geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/i18n/CliStrings.java

```diff
@@ -63,65 +59,67 @@
-  public static final String TOPIC_GEMFIRE_MANAGER = "Manager";
-  public static final String TOPIC_GEMFIRE_MANAGER__DESC = "The Manager is a member which has the additional role of a managing & monitoring the GemFire distributed system.";
-  public static final String TOPIC_GEMFIRE_STATISTICS = "Statistics";
-  public static final String TOPIC_GEMFIRE_STATISTICS__DESC = "Every application and server in a vFabric GemFire distributed system can be configured to perform statistical data collection for analysis.";
-  public static final String TOPIC_GEMFIRE_LIFECYCLE = "Lifecycle";
-  public static final String TOPIC_GEMFIRE_LIFECYCLE__DESC = "Launching, execution and termination of GemFire cluster members such as servers and locators.";
-  public static final String TOPIC_GEMFIRE_M_AND_M = "Management-Monitoring";
-  public static final String TOPIC_GEMFIRE_M_AND_M__DESC = "The management of and monitoring of GemFire systems using GemFire tools, such as vFabric GemFire Pulse or VSD or Data Browser, and JConsole, which is provided with the JDK(TM)";
-  public static final String TOPIC_GEMFIRE_DATA = "Data";
-  public static final String TOPIC_GEMFIRE_DATA__DESC = "User data as stored in regions of the GemFire distributed system.";
-  public static final String TOPIC_GEMFIRE_CONFIG = "Configuration";
-  public static final String TOPIC_GEMFIRE_CONFIG__DESC = "Configuration of vFabric GemFire Cache & Servers/Locators hosting the Cache.";
-  public static final String TOPIC_GEMFIRE_FUNCTION = "Function Execution";
-  public static final String TOPIC_GEMFIRE_FUNCTION__DESC = "The function execution service provides solutions for these application use cases: \n\tAn application that executes a server-side transaction or carries out data updates using the GemFire distributed locking service. \n\tAn application that needs to initialize some of its components once on each server, which might be used later by executed functions. Initialization and startup of a third-party service, such as a messaging service. \n\tAny arbitrary aggregation operation that requires iteration over local data sets that can be done more efficiently through a single call to the cache server. \n\tAny kind of external resource provisioning that can be done by executing a function on a server.";
-  public static final String TOPIC_GEMFIRE_HELP = "Help";
-  public static final String TOPIC_GEMFIRE_HELP__DESC = "Provides usage information for gfsh & its commands.";
-  public static final String TOPIC_GEMFIRE_DEBUG_UTIL = "Debug-Utility";
-  public static final String TOPIC_GEMFIRE_DEBUG_UTIL__DESC = "Debugging aids & utilities to use with vFabric GemFire.";
+  public static final String DEFAULT_TOPIC_GEODE = "Geode";
+  public static final String DEFAULT_TOPIC_GEODE__DESC = "Apache Geode is a distributed data management platform providing dynamic scalability, high performance and database-like persistence.";
```

Comment:
When referencing the projecting as 'Apache Geode', I think we should be using 'Apache Geode (incubating)' until Geode is a top-level Apache project.